### DISCUSSION TITLE
How can I change to the Horizon theme?

### DISCUSSION THREAD

**Discussion by thelaith** [swellstores/community#110] [Help] [answered]
Hey Swell, 

I've been trying to install the Horizon theme off Github, but it's proving a bit confusing. Is there a way to install it through the Swell Admin UI?

**Comment by cristianstanescu** [comment, marked as answer]
Unfortunately we don't have a way yet to install Horizon via Swell Admin. We are actively working on improving the experience and making this feature available, can't say exactly when we'll launch it though.

Meanwhile, which part(s) do you find confusing, maybe we can help or improve documentation together?

**Comment by thelaith** [comment]
Hey!

I tried installing it via vercel, but I reached the localhost stage, and it didn't seem to work - Can't quite remember the editor error - but is it not possible to add the option to replace the files through Horizon?

**Comment by thelaith** [comment]
I couldn't manage to get the site up on vercel on the proper domain name, and had a seperate "testing" domain name to deal with. I was under the impression that Horizon was right around the corner to be released so that we could enable that as the default theme and configure it graphically in the UI. Is there a step by step video guide that shows you what steps to take to get it to that stage?
